src/gui/screen_menu_steel_sheets.cpp

```diff
@@ -122,11 +122,12 @@ class SheetProfileMenuScreenT : public SheetProfileMenuScreen {
         case profile_action::Select:
             log_debug(GUI, "MI_SHEET_SELECT");
             sheet_profile_select(Index::value);
-            marlin_set_z_offset(variant8_get_flt(eeprom_get_var(EEVAR_ZOFFSET)));
+            marlin_set_z_offset(variant8_get_flt(eeprom_get_z_offset()));
             break;
         case profile_action::Calibrate:
             log_debug(GUI, "MI_SHEET_CALIBRATE");
             sheet_profile_select(Index::value);
+            marlin_set_z_offset(variant8_get_flt(eeprom_get_z_offset()));
```

Comment:
eeprom_get_z_offset() returns float and not variant8_t

Reply:
not needed at all - just reads from eeprom than writes same value at same place, removed